install-air-gap.hbs.md

```diff
@@ -234,8 +261,9 @@ profile: full
 ceip_policy_disclosed: true
 buildservice:
   kp_default_repository: "REPOSITORY"
-  kp_default_repository_username: "REGISTRY-USERNAME" # Takes the value from the shared section above by default, but can be overridden by setting a different value.
-  kp_default_repository_password: "REGISTRY-PASSWORD" # Takes the value from the shared section above by default, but can be overridden by setting a different value.
+  kp_default_repository_secret: # Takes the value from the shared section above by default, but can be overridden by setting a different value.
```

Comment:
Is it applicable (or worth including) this comment about the shared section on [line 413](https://github.com/pivotal/docs-tap/pull/2345/files#diff-ce6aea12f7588bb7a07de306c853a2071b7697c664e27a74f972ce488dac9cb4R413)

Reply:
ya we can